kubernetes/jobs/datalake-data-export/qa/cronjob.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  labels:
+    app: {{SERVICE_NAME}}
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+spec:
+  schedule: "45 6 * * *"
+  suspend: true
```

Comment:
this is fine, just a note to us: if we plan to have differences in manifests across environments we should probably think about a strategy for automations